src/main/java/com/metamx/common/io/smoosh/FileSmoosher.java

```diff
@@ -252,7 +263,7 @@ private Outer getNewCurrOut() throws FileNotFoundException
     final int fileNum = outFiles.size();
     File outFile = makeChunkFile(baseDir, fileNum);
     outFiles.add(outFile);
-    return new Outer(fileNum, new BufferedOutputStream(new FileOutputStream(outFile)), maxChunkSize);
+    return new Outer(fileNum, new FileOutputStream(outFile).getChannel(), maxChunkSize);
```

Comment:
I think FileOutputStream should be created with try-with-resources, just in case.